libcamera/src/camera_manager.rs

```diff
@@ -33,6 +34,15 @@ impl CameraManager {
     pub fn cameras(&self) -> CameraList<'_> {
         unsafe { CameraList::from_ptr(NonNull::new(libcamera_camera_manager_cameras(self.ptr.as_ptr())).unwrap()) }
     }
+
+    /// Set the log level.
+    pub fn log_set_level(&self, category: &str, level: LoggingLevel) {
```

Comment:
Is there a reason why `log_set_level` is part of the `CameraManager`, while other logging methods are global under `logging` module?

Reply:
I mentioned that in the PR introduction message :)
> log_set_level() is implemented as the CameraManager instance method because in my testing I was only be able to see the effect of this function after the creation of CameraManager, any calls before that seem to be ignored.

Reply:
Ah my bad, it's been 3 weeks since I read that 😅